data-avro/src/main/java/com/linkedin/data/avro/PegasusUnionToAvroRecordConvertCallback.java

```diff
@@ -1,8 +1,8 @@
 package com.linkedin.data.avro;
 
-import com.google.common.base.CaseFormat;
 import com.linkedin.data.Data;
 import com.linkedin.data.DataMap;
+import com.linkedin.data.avro.util.guava.CaseFormat;
```

Comment:
The only place we use CaseFormat is to convert from lowerCamelCase to UpperCamelCase for field names which will always be ascii.
So should we avoid copying the utilities and just write a utility fn that replaces first char with uppercase?